not_mvp/1224-user_administration/1313-administrator_opens_user_administration_part_ii.feature

```diff
@@ -0,0 +1,19 @@
+    @1313
+    Scenario: Administrator opens User Administration (Part II)
+        Given that the user is logged in as Administrator
+        When they click the menu item User Administration
+        Then they see the User Administration page
+        And they see the Sections
+            | Institution Administrators |
+            | Curators                   |
+            | Editors                    |
```

Comment:
Out of sync with Part I. Creators are missing.

Reply:
replaced description with the same format as part 1